cpp/src/io/parquet/page_string_utils.cuh

```diff
@@ -107,4 +109,18 @@ __device__ void block_excl_sum(size_type* arr, size_type length, size_type initi
   }
 }
 
+/**
```

Comment:
Still debating myself if this miniscule inlined function should be removed and we should just embrace code duplication in page_string_decode.cu and page_delta_decode.cu.